packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStop.ts

```diff
@@ -14,7 +17,7 @@ import { showError } from './commandUtils';
 const sfdxCoreExports = vscode.extensions.getExtension(
   'salesforce.salesforcedx-vscode-core'
 )!.exports;
-const { notificationService, telemetryService } = sfdxCoreExports;
+const { telemetryService } = sfdxCoreExports;
```

Comment:
Do we have another work item to finish replacing the rest of the telemetry service usage ?

Reply:
Created W-8290977 for replacing the rest of telemetryService